addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11636,13 +11636,13 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#21472"
-msgid "Include All Seasons and Specials"
+msgid "Include all seasons and specials"
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Include All Seasons and Specials" with label #21472
 #: system/settings/settings.xml
 msgctxt "#21473"
-msgid "Whether or not to consider All Seasons and Special items in the unwatched item selection."
+msgid "Whether or not to consider all seasons and special items in the unwatched item selection."
```

Comment:
It's not because I like to treat them as nouns, but it's because they are actually a reference to other parts in Kodi (just go to a tv show in TV-Shows section and you'll have the "All seasons" node), which is with everything lowercase and/or without quotes not obvious anymore. Please read my suggested change from above again.